apps/meteor/app/emoji/client/helpers.ts

```diff
@@ -149,7 +149,10 @@ export const removeFromRecent = (emoji: string, recentEmojis: string[], setRecen
 	setRecentEmojis(recentEmojis);
 };
 
-export const updateRecent = (recentList: string[]) => {
+export const updateRecent = (recentList: string | string[]) => {
+	if (!Array.isArray(recentList)) {
```

Comment:
I'm wondering how we pass string sometimes and ts doest not complain, would you mind do dig a bit more and find the root cause? this solves the consequence but i'm afraid does not solve the real isse

Reply:
hey @ggazzo !
This refactor might not be needed anymore because only arrays are passed now due to recent refactorings.
But i have found out potential bugs.
Here we are calling updateRecent with hard coded string value 'rocket'. Instead, we should call updateRecent with the value of the emoji which was recently deleted. :
https://github.com/RocketChat/Rocket.Chat/blob/5fddfcc99ea6d17eb63cd33772dc7388a1d7fba8/apps/meteor/app/emoji-custom/client/lib/emojiCustom.ts#L52
same here:
https://github.com/RocketChat/Rocket.Chat/blob/5fddfcc99ea6d17eb63cd33772dc7388a1d7fba8/apps/meteor/app/emoji-custom/client/lib/emojiCustom.ts#L97

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/33808
This pr fixes the above mentioned issue.
Closing the current PR.